src/containers/Sidebar.js

```diff
@@ -37,7 +51,45 @@ export class Sidebar extends Component {
           </li>
         );
       }
+    }).filter(Boolean);
+
+    const accordionClasses = classnames({
+      'accordion-label': true,
```

Comment:
You don't need to pass true for classes to be added directly. You can do this;
```js
classnames("accordion-label", { collapsed: .. })
```

Reply:
Thanks. I didn't know this shorthand. :+1: